xos/configurations/common/mydeployment.yaml

```diff
@@ -16,9 +16,15 @@ topology_template:
     m1.small:
       type: tosca.nodes.Flavor
 
+    m1.xlarge:
+      type: tosca.nodes.Flavor
+
     MyDeployment:
       type: tosca.nodes.Deployment
       requirements:
+          - m1.xlarge:
+             node: m1.large
```

Comment:
I guess this is just a typo, but it doesn't much sense I will correct it.

On Tue, May 10, 2016 at 10:02 PM, pingping-lin <notifications@github.com>
wrote:

> In xos/configurations/common/mydeployment.yaml
> <https://github.com/open-cloud/xos/pull/141#discussion_r62732101>:
>
> >      MyDeployment:
> >        type: tosca.nodes.Deployment
> >        requirements:
> > +          - m1.xlarge:
> > +             node: m1.large
>
> is this m1.large should be m1.xlarge?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/open-cloud/xos/pull/141/files/957b8f0b6ba1390c8bdf7c475d431918155ddf69#r62732101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGcQ9GnJByLHibLLXzhP857DY0wr8DCuks5qANZXgaJpZM4IPTEs>
> .
>